metrics/core/src/main/java/tech/pegasys/pantheon/metrics/PantheonMetricsSystem.java

```diff
@@ -12,11 +12,14 @@
  */
 package tech.pegasys.pantheon.metrics;
 
-import java.util.Optional;
+import tech.pegasys.pantheon.plugin.services.MetricsSystem;
+import tech.pegasys.pantheon.plugin.services.metrics.MetricCategory;
 
-public interface MetricCategory {
+import java.util.stream.Stream;
 
-  String getName();
+public interface PantheonMetricsSystem extends MetricsSystem {
```

Comment:
Can we call this something other than `Pantheon...`.  Maybe `ObservableMetricsSystem` or even `MetricsSource`.